pagopa/io-app
# fix: [IABT-1408,IOAPPCOM-17] Fixed toggles on Profile Preferences for Push Notifications Opt-in

## Short description
This PR fixes the toggles on profile preferences for push notifications opt-in where, starting with an opposite toggle configuration (_i.e._, one on and one off), enabling the disabled toggle caused the other toggle to turn off.

## List of changes proposed in this pull request
- Edited `createOrUpdateProfileSaga` in `ts/sagas/profile.ts`to always send both `push_notifications_content_type` and `reminder_status` values (please note that, when their value is undefined, the latter is not included in the request payload. This is vital in order not to send user preferences for the opt-in before the user has landed on the Opt-in view, during the onboarding).

## How to test
Make sure to use an application configuration that points to the production environment and go through the onboarding or opt-in process (or, if you have already opted-in, just start the application). Navigate to the preferences screen for the Push Notifications, make sure to have one toggle on and one toggle off and then proceed by changing the off-toggle to on. The desired behaviour is that a toggle does not change the other one's value.
